### DISCUSSION TITLE
Deno 1.0 support

### DISCUSSION THREAD

**Discussion by pierreyves-lebrun** [prisma/prisma#2436] [Q&A] [answered]
Deno 1.0 just got released today, do you have any plan for supporting it?

https://deno.land/v1

**Comment by pierreyves-lebrun** [comment, marked as answer]
Apologies for the duplicate question, I hadn’t seen https://github.com/prisma/prisma/discussions/2415

**Comment by ludralph** [comment]
Hi @pierreyves-lebrun,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
